frontend/src/features/public-form/components/SectionSidebar/SectionSidebar.tsx

```diff
@@ -21,11 +21,12 @@ import { useFormSections } from '../FormFields/FormSectionsContext'
 
 import { SidebarLink } from './SidebarLink'
 
-export const SectionSidebar = (): JSX.Element => {
+export const SectionSidebar = (): JSX.Element | null => {
```

Comment:
Why is this needed? It doesn't look like the changeset can make `SectionSidebar()` return `null`

Reply:
Oops that was when I used null initially and I forgot to remove it. Sincere apologies @timotheeg

Reply:
No worries at all. Remove it in a new commit and we can merge!

Reply:
removed! thank you very much!